### PR TITLE
Micah

### PR DESCRIPTION
**Pull request by mw1510**

### CODE REVIEW

File: frontend/src/components/sitemanager/SiteManagerHomepage.js [hunk 1]
```diff
@@ -113,7 +240,14 @@ class SiteManagerHomepage extends Component {
...
                             <PopupMealTotals weekArr= {weekArr} day={4} showModal={this.handleOpenModal}/>
                         </div>
-                        <button className="route" style={{marginTop: 20,width: 300}} onClick={this.printDocument}>Print Meal Totals</button>
+                        <div className = "confirmation-buttons" style={{ display:'flex', marginTop: 20}}>
+                            <h3>Driver Routes For: </h3>
+                            <button className="route" onClick={() => this.printDocument("SLO", "M", 0)}>Monday</button>
+                            <button className="route" onClick={() => this.printDocument("SLO", "T", 1)}>Tuesday</button>
+                            <button className="route" onClick={() => this.printDocument("SLO", "W", 2)}>Wednesday</button>
+                            <button className="route" onClick={() => this.printDocument("SLO", "Th", 3)}>Thursday</button>
+                            <button className="route" onClick={() => this.printDocument("SLO", "F", 4)}>Friday</button>
```
Comment (EmilyGavrilenko) [inline frontend/src/components/sitemanager/SiteManagerHomepage.js]: You're hardcoding "SLO" as the site, replace with localStorage.get("site")

File: backend/src/meals.js [hunk 2]
```diff
@@ -99,6 +99,80 @@ async function existsMeal(client, week) {
...
+    // takes in these parameters from the front end.
+    // site is used to search and day which is a string M, T, W, Th, F
+    let {site, day} = req.body
+    
+    // this calls the Clients database and searches for matches to the site
+    // if there is an error it prints in the backend console 
+    // WARNING: danger of finding all clients with the same site
+    // if someone puts in "SLO" and another "San Luis Obispo" they will not be 
+    // found in the same list. This function will sort by SLO
+    Client.find({site: site}, function (err, clients) {
```
Comment (EmilyGavrilenko) [inline backend/src/meals.js:124]: Fetch data from meals database, not the client database. Route totals will change week by week.

File: frontend/src/components/sitemanager/SiteManagerHomepage.js (hunk 1 above)
Comment (mw1510) [reply frontend/src/components/sitemanager/SiteManagerHomepage.js]: Updated. Thanks, wasn't sure where to get that information

**Review by EmilyGavrilenko — COMMENTED** [review]
Please update to meals database

File: backend/src/meals.js [hunk 3]
```diff
@@ -99,6 +99,110 @@ async function existsMeal(client, week) {
     return output
 }
 
+async function findClientMeal(client, week) {
+  let data
+  await Meal.findOne({clientID: client._id, startDate: week}, function(err, meal) {
+    if (err) {
+      console.log("error")
```
Comment (EmilyGavrilenko) [inline backend/src/meals.js:106]: Don't display clients with no meal data. Right now it's unable to handle past weeks. To test, try to print a week in April

**Review by EmilyGavrilenko — APPROVED** [review]
Update to use meals data for foodDays and update button widths, approving for now

**Review by EmilyGavrilenko — APPROVED** [review]
Looks good

**Pull request merged**